packages/playground/resolve/index.html

```diff
@@ -107,6 +116,18 @@ <h2>resolve package that contains # in path</h2>
   import deepPath from 'resolve-exports-path/deep.json?url'
   text('.exports-deep-query', deepPath)
 
+  // deep import SVG w/ exports w/ query
+  import deepSvgPath from 'resolve-exports-path/deep.svg?url'
+  text('.exports-deep-svg-query', deepSvgPath)
```

Comment:
This one works 👍 So we can rule out a problem of it only working with `.js` or `.json` files